src/platforms/common/swdptap.c

```diff
@@ -70,15 +70,16 @@ static void swdptap_turnaround(const swdio_status_t dir)
 		SWDIO_MODE_FLOAT();
 	} else {
 		gpio_clear(SWCLK_PORT, SWCLK_PIN);
-		for (volatile uint32_t counter = target_clk_divider + 1; counter > 0; --counter)
-			continue;
 	}
```

Comment:
Please eliminate the braces from the else block (can't for the if due to the macro, but can for the else)